deploy/csi-driver/000-namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ovirt-csi-driver
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    name: ovirt-csi-driver
+    openshift.io/run-level: "1"
```

Comment:
remove run-level completely, otherwise this bypasses security checks. I hope all the hostpath will not fail because of that....

Reply:
ACK